rollup.config.js

```diff
@@ -55,102 +67,36 @@ export default {
-            'node_modules/carbon-components/scss/globals/scss/_helper-mixins.scss',
-            'node_modules/carbon-components/scss/globals/scss/_import-once.scss',
-            'node_modules/carbon-components/scss/globals/scss/_layer.scss',
-            'node_modules/carbon-components/scss/globals/scss/_layout.scss',
-            'node_modules/carbon-components/scss/globals/scss/_mixins.scss',
-            'node_modules/carbon-components/scss/globals/scss/_motion.scss',
-            'node_modules/carbon-components/scss/globals/scss/_spacing.scss',
-            'node_modules/carbon-components/scss/globals/scss/_theme-tokens.scss',
-            'node_modules/carbon-components/scss/globals/scss/_theme.scss',
-            'node_modules/carbon-components/scss/globals/scss/_tooltip.scss',
-            'node_modules/carbon-components/scss/globals/scss/_typography.scss',
-            'node_modules/carbon-components/scss/globals/scss/_vars.scss',
-          ],
-          dest: 'lib/scss/globals/scss',
+          src: 'src/globals',
+          dest: 'lib/scss',
+        },
+
+        // Sass components
+        {
+          src: 'src/components/**/*.scss',
+          dest: 'lib/scss/components',
+          rename: (name, extension) => sanitizeAndCamelCase(name, extension),
         },
-        { src: 'node_modules/carbon-components/scss/globals/grid', dest: 'lib/scss/globals' },
+
+        // Carbon components proxy styles
         {
           src: [
-            // ------------------------------
-            // Carbon Components Proxy Styles
-            // ------------------------------
-            // These are listed individually so that we can easily override
-            // these with our own versions of these files.
-            'node_modules/carbon-components/scss/components/button',
-            'node_modules/carbon-components/scss/components/copy-button',
-            'node_modules/carbon-components/scss/components/file-uploader',
-            'node_modules/carbon-components/scss/components/checkbox',
-            'node_modules/carbon-components/scss/components/combo-box',
-            'node_modules/carbon-components/scss/components/radio-button',
-            'node_modules/carbon-components/scss/components/toggle',
-            'node_modules/carbon-components/scss/components/search',
-            'node_modules/carbon-components/scss/components/select',
-            'node_modules/carbon-components/scss/components/text-input',
-            'node_modules/carbon-components/scss/components/text-area',
-            'node_modules/carbon-components/scss/components/number-input',
-            'node_modules/carbon-components/scss/components/form',
-            'node_modules/carbon-components/scss/components/link',
             'node_modules/carbon-components/scss/components/list-box',
             'node_modules/carbon-components/scss/components/list',
             'node_modules/carbon-components/scss/components/data-table',
```

Comment:
@davidicus There are 5 component files we're still directly copying and importing from `node_modules`

1. `'node_modules/carbon-components/scss/components/list-box'`
1. `'node_modules/carbon-components/scss/components/list'`
1. `'node_modules/carbon-components/scss/components/data-table'`
1. `'node_modules/carbon-components/scss/components/skeleton'`
1. `'node_modules/carbon-components/scss/components/pagination-nav'`

I'm not sure what to do with these. They're either not exported directly from carbon-components-react, or we have our own version (data-table). I could create a folder and file for them like I have the others, but as we're not exporting them, this feels odd. Thoughts?

Reply:
I think its probably a good idea to create folder and file for each and import them there as its consistent and if we ever do export them as modules they will have the infrastructure in place. The data-table partial can probably be brought into our table component no? I know we export both carbon tables and our own but the styles on top of these shouldnt clash

Reply:
👍 Will add folders for these so we're not copying directly from `node_modules`